tools/update-metadata.lua

```diff
@@ -65,38 +48,315 @@ local function getLocales()
+  <url type="contribute">https://koreader.rocks/doc/</url>
+
+  <branding>
+    <color type="primary" scheme_preference="light">#ffffff</color>
+    <color type="primary" scheme_preference="dark">#303030</color>
+  </branding>
+
+  <requires>
+    <memory>128</memory>
+    <display_length>360</display_length>
+  </requires>
+
+  <supports>
+    <control>pointing</control>
+    <control>keyboard</control>
+    <control>gamepad</control>
+  </supports>
+
+  <recommends>
+    <memory>1024</memory>
+    <display_length compare="ge">600</display_length>
+    <internet>always</internet>
+    <control>touch</control>
+  </recommends>
+
+  <provides>
+    <mediatype>application/epub+zip</mediatype>
+    <mediatype>application/fb2</mediatype>
+    <mediatype>application/fb3</mediatype>
+    <mediatype>application/msword</mediatype>
+    <mediatype>application/oxps</mediatype>
+    <mediatype>application/pdf</mediatype>
+    <mediatype>application/rtf</mediatype>
+    <mediatype>application/tcr</mediatype>
+    <mediatype>application/vnd.amazon.mobi8-ebook</mediatype>
+    <mediatype>application/vnd.comicbook+tar</mediatype>
+    <mediatype>application/vnd.comicbook+zip</mediatype>
+    <mediatype>application/vnd.ms-htmlhelp</mediatype>
+    <mediatype>application/vnd.openxmlformats-officedocument.wordprocessingml.document</mediatype>
+    <mediatype>application/vnd.palm</mediatype>
+    <mediatype>application/x-cbz</mediatype>
+    <mediatype>application/x-chm</mediatype>
+    <mediatype>application/x-fb2</mediatype>
+    <mediatype>application/x-fb3</mediatype>
+    <mediatype>application/x-mobipocket-ebook</mediatype>
+    <mediatype>application/x-tar</mediatype>
+    <mediatype>application/xhtml+xml</mediatype>
+    <mediatype>application/xml</mediatype>
+    <mediatype>application/zip</mediatype>
+    <mediatype>image/djvu</mediatype>
+    <mediatype>image/gif</mediatype>
+    <mediatype>image/jp2</mediatype>
+    <mediatype>image/jpeg</mediatype>
+    <mediatype>image/jxr</mediatype>
+    <mediatype>image/png</mediatype>
+    <mediatype>image/svg+xml</mediatype>
+    <mediatype>image/tiff</mediatype>
+    <mediatype>image/vnd.djvu</mediatype>
+    <mediatype>image/vnd.ms-photo</mediatype>
+    <mediatype>image/x-djvu</mediatype>
+    <mediatype>image/x-portable-arbitrarymap</mediatype>
+    <mediatype>image/x-portable-bitmap</mediatype>
+    <mediatype>text/html</mediatype>
+    <mediatype>text/plain</mediatype>
+  </provides>
+
+  <launchable type="desktop-id">rocks.koreader.KOReader.desktop</launchable>
+
+  <content_rating type="oars-1.1"/>
+]],
+}
+
+local function summary(lang)
+    local lang = lang or "en"
+    _.changeLang(lang)
+    return _(metadata.summary)
+end
+
+local function htmlDescription(lang)
+    local lang = lang or "en"
+    local desc = metadata.desc
+    local t = {}
+    _.changeLang(lang)
+    for i, v in ipairs (desc.paragraphs) do
+        table.insert(t, "<p>" .. _(v) .. "</p>")
```

Comment:
As stated, I'd prefer it like this so it's future-proof:
```suggestion
        table.insert(t, "<p>" .. util.htmlEscape(_(v)) .. "</p>")
```